msg/CartesianTrajectory.msg

```diff
@@ -0,0 +1,3 @@
+string eef_frame
```

Comment:
could you add a comment at top explaining the intention of this file, and the others?

Reply:
Added everywhere